storage/src/vespa/storage/persistence/filestorage/filestorhandlerimpl.cpp

```diff
@@ -78,7 +78,10 @@ FileStorHandlerImpl::FileStorHandlerImpl(uint32_t numThreads, uint32_t numStripe
     _component.registerMetricUpdateHook(*this, framework::SecondTime(5));
 }
 
-FileStorHandlerImpl::~FileStorHandlerImpl() = default;
+FileStorHandlerImpl::~FileStorHandlerImpl()
+{
+    waitUntilNoLocks();
```

Comment:
Consider not releasing lock before calling notify_all() at end of` FileStoreHandlerImpl::Stripe::release()`